x/wasm/client/cli/gov_tx.go

```diff
@@ -351,3 +352,133 @@ func ProposalClearContractAdminCmd() *cobra.Command {
 	cmd.Flags().String(flagProposalType, "", "Permission of proposal, types: store-code/instantiate/migrate/update-admin/clear-admin/text/parameter_change/software_upgrade")
 	return cmd
 }
+
+func ProposalPinCodesCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "pin-codes [code-ids]",
+		Short: "Submit a pin code proposal for pinning a code to cache",
+		Args:  cobra.ExactArgs(1),
```

Comment:
```suggestion
		Args:  cobra.MinimumNArgs(1),
```
I assume you want to support multiple ids as described in the "Use" line